scheduler/orchestrator/notifications.py

```
@@ -198,6 +247,155 @@ def send_sync_complete(
                 "text": f"{prefix}:cloud: `{hub}` data synced to S3{dur_str}"
             })
 
+    # =========================================================================
+    # Hub-complete success notification (→ #tech with @here)
+    # =========================================================================
+
+    def send_hub_complete_success(
+        self,
+        hub: str,
+        harvest_count: int | None = None,
+        mapping_attempted: int | None = None,
```

Ruff (0.15.2)

[warning] Unused method argument: `mapping_attempted`

(ARG002)